spotify_confidence/analysis/frequentist/confidence_computers/t_test_computer.py

```diff
@@ -20,77 +24,82 @@
 )
 
 
-class TTestComputer(object):
```

Comment:
TTestComputer, ZTestComputer etc are not classes anymore, to enable threading.

Reply:
And instead of working on individual rows, they work on dataframes, typically containing all rows from a metric/segment.